app/public/humanState.js

```diff
@@ -43,11 +43,242 @@ const HUMAN_STATE = (() => {
             currentToolButton.classList.add("selected");
         }
     }
+
+    function initiateMiniGame(tool){
```

Comment:
We don't need to add the raw html, css, and javascript to the page, we can just display/undisplay the game using display: none and display: block. The html and css can be put onto the html page for the human player. The game can be reset with functions.